pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -112,6 +116,14 @@ func (c *InstallerController) WithCerts(certDir string, certConfigMaps, certSecr
 	return c
 }
 
+// WithStartupMonitorSupport sets the predicate startupMonitorDeployed called on every sync
+// to know whether startup monitor is deployed, and the state machine can expect that the
+// startup-monitor acknowledged ready operands.
+func (c *InstallerController) WithStartupMonitorSupport(startupMonitorDeployed func() (bool, error)) *InstallerController {
+	c.startupMonitorDeployed = startupMonitorDeployed
```

Comment:
is `startupMonitorDeployed` going to tell that the startup monitor is installed on a node?

Reply:
yes, every time an installer is created.